src/Cody.VisualStudio.Tests/ChatLoggedBasicTests.cs

```diff
@@ -1,57 +1,83 @@
-using EnvDTE;
-using Microsoft.VisualStudio.Shell;
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace Cody.VisualStudio.Tests
 {
     public class ChatLoggedBasicTests: PlaywrightInitializationTests
     {
-        [VsFact(Version = VsVersion.VS2022)]
+        public ChatLoggedBasicTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        //[VsFact(Version = VsVersion.VS2022)]
```

Comment:
what is this for?

Reply:
This is for XUnit logger, which is passing calls to our Logger. Thanks to it we have nice, detailed logs https://github.com/sourcegraph/cody-vs/actions/runs/10814320609/job/30000502786

![image](https://github.com/user-attachments/assets/94d2e28c-3da3-4579-9a1c-537948a9b2a2)